File: 3110. Score of a String.php
class Solution {
    function scoreOfString($s) {
    $res = 0;
    for ($i = 0; $i < strlen($s) - 1; $i++) {
        $res += abs(ord($s[$i]) - ord($s[$i + 1]));
    }
    return $res; 
    }
}

File: 3110. Score of a String.rs
impl Solution {
    pub fn score_of_string(s: String) -> i32 {
    let mut res: i32 = 0;
    for i in 0..s.len()-1 {
        res += (s.as_bytes()[i] as i32 - s.as_bytes()[i+1] as i32).abs();
    }
    res
    }
}
